lib/chef_zero/server.rb

```diff
@@ -111,7 +111,7 @@ module ChefZero
   class Server
 
     DEFAULT_OPTIONS = {
-      :host => '127.0.0.1',
+      :host => ['127.0.0.1'],
```

Comment:
I was going to add ::1 by default, but then... figured that'd probably break something somewhere... I guess I was right. :)